src/Setup/Assemblies.wxi

```diff
@@ -18,6 +18,10 @@
       <File Source="$(var.BinDir)\Microsoft.R.Interpreters.dll" />
       <File Source="$(var.BinDir)\Microsoft.Data.ConnectionUI.dll" />
       <File Source="$(var.BinDir)\Microsoft.Data.ConnectionUI.Dialog.dll" />
+      <File Source="$(var.BinDir)\Microsoft.SqlServer.Dac.dll" />
```

Comment:
Don't forget to add those assemblies to `Create-FilesSwr.ps1` in low-impact installer.